frontend/src/metabase/api/tags.ts

```diff
@@ -14,5 +14,8 @@ export function providesList<
     : [listTag];
 }
 
-export const tagTypes = [];
+export const API_KEY_TAG = "ApiKey" as const;
+export const API_KEY_LIST_TAG = getListTag(API_KEY_TAG);
```

Comment:
WDYT about defining these in the api-keys file instead?

Reply:
This file was born out of a circular import issue 😢

Reply:
I think it's ultimately safer to do this in the case we may have two api definitions try to invalidate some part of each others cache. Also since tags need to be provided to the createApi call, you can't get away from editing two files.